features/FEATURE_BLE/targets/TARGET_CORDIO/CordioPalGenericAccessService.h

```diff
@@ -19,69 +19,6 @@ class GenericAccessService : public ::ble::pal::GenericAccessService {
 
     virtual ~GenericAccessService() { }
 
-    virtual ble_error_t get_device_name_length(uint8_t& length) {
```

Comment:
Can we keep these functions in Cordio, it will be easier for us if they are here to enable publicly the feature.

Reply:
Those functions use some deprecated types (e.g. `GapAdvertisingData` which is replaced by `AdvertisingDataBuilder`). Can we use macros (e.g. `#if 0`) to disable them, and fix them once we reenable the features?

Reply:
that's ok if remove the code with `#if 0`. Please make sure to have a ticket open for these.